gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/component/GbGradeTable.html

```diff
@@ -149,7 +149,7 @@
                         <li><a href="javascript:void(0);" class="gb-hide-column" role="menuitem" data-assignmentid="${assignmentId}"><wicket:message key="assignment.option.hide" /></a></li>
                         <li><a href="javascript:void(0);" class="gb-message-students" role="menuitem" data-assignment-id="${assignmentId}"><wicket:message key="label.submission-messager.title" /></a></li>
                         {if !externallyMaintained && settings.isUserAbleToEditAssessments}
-                            <li><a href="javascript:void(0);" class="gb-delete-item" role="menuitem"><span class="text-danger"><wicket:message key="delete.label" /></span></a></li>
+                            <li><a href="javascript:void(0);" class="gb-delete-item" role="menuitem"><span class="sak-banner-error-inline"><wicket:message key="delete.label" /></span></a></li>
```

Comment:
GB categories related changes needed because the Bootstrap mixins didn't support CSS Custom Properties.